src/coord/cartesian/Axis2D.ts

```diff
@@ -39,7 +40,7 @@ interface Axis2D {
      */
     toGlobalCoord(coord: number): number;
 }
-class Axis2D extends Axis {
+class Axis2D extends Axis implements JitterStorable {
```

Comment:
I didn't see any usage of the `jitterStore` member of the `JitterStorable` interface. It can be removed safely.